lib/brewdler/utils.rb

```diff
@@ -8,11 +8,28 @@ def self.system cmd, *args
     $?.success?
   end
 
-  def self.brew_installed?
-    system("brew --version &>/dev/null")
+  def brew_installed?
+    Kernel.system("brew --version &>/dev/null")
   end
 
-  def self.cask_installed?
-    system("brew cask --version &>/dev/null")
+  def cask_installed?
+    Kernel.system("brew cask --version &>/dev/null")
+  end
+
+  def brewfile
+    File.read(Dir['{*,.*}{B,b}rewfile'].first.to_s)
+  rescue Errno::ENOENT
+    raise "No Brewfile found"
+  end
+
+  def getc  # NOTE only tested on OS X
+    Kernel.system "/bin/stty raw -echo"
+    if STDIN.respond_to?(:getbyte)
+      STDIN.getbyte
+    else
+      STDIN.getc
+    end
+  ensure
+    Kernel.system "/bin/stty -raw echo"
   end
```

Comment:
> I think you could just always require --force

Then, what is the point of using `--force`. IMHO, I prefer current design. It has both interactive and non-interactive features. So everyone is happy.